.werft/util/gcloud.ts

```diff
@@ -33,3 +35,28 @@ export async function deleteExternalIp(phase: string, name: string, region = "eu
 function getExternalIp(name: string, region = "europe-west1") {
     return exec(`gcloud compute addresses describe ${name} --region ${region}| grep 'address:' | cut -c 10-`, { silent: true }).trim();
 }
+
+export function createDNSRecord(domain: string, dnsZone: string, IP: string, slice: string) {
+    const werft = getGlobalWerftInstance()
+    werft.log(slice, `Creating DNS record. domain: ${domain} DNSZone: ${dnsZone} IPAddress: ${IP}`)
+
+    const dnsClient = new DNS({
+        projectId: 'gitpod-dev',
+        keyFilename: GCLOUD_SERVICE_ACCOUNT_PATH
+    })
+    const zone = dnsClient.zone(dnsZone)
+    const record = new Record(zone, 'a', {
+        name: domain,
+        ttl: 300,
+        signatureRrdatas: [IP],
+        data: IP
```

Comment:
The error you're getting sounds like a bug in @google-cloud/dns to be honest. You could try to remove the `signatureRrdatas` from here just because that's the only array you're passing in